etabotsite/etabotsite/settings.py

```diff
@@ -41,9 +45,49 @@
     logging.warning('cannot load custom_settings.json due to "{}"'.format(
         e))
 
+CUSTOM_SETTINGS = custom_settings
+PROD_HOST_URL = prod_host_url
+
+# Determine if we are running on local mode or production mode
+# This is based on custom settings config.
+# This makes control of prod environment more controlable cross-platform
+LOCAL_MODE = custom_settings.get("LOCAL_MODE")
+
+
+# System email settings
+# Must be gathered before logging is initiatied to use for email alerts!
+
+SYS_DOMAIN = local_host_url if LOCAL_MODE else prod_host_url
+ADMIN_EMAILS = [""]
```

Comment:
should this be list of one empty string or empty list by default?

Reply:
I set it to an empty list now, I also changed the ADMIN email to be an if statement like above so you don't need to change prod settings if you don't want. It will just default to an empty list.